src/store/index.js

```diff
@@ -1,5 +1,6 @@
 import Vue from 'vue'
-import Vuex from 'vuex'
+// import Vuex from 'vuex'
+import Vuex from 'vuex/dist/vuex.common' // Why this: `Render()` in Testing Library uses common js
```

Comment:
Interesting solution. 
So the testing library use `instanceof` to check whether the store passed into is an instance of `Vuex` under the hood. In this case, if we use the ESM version of 'Vuex` here, it causes errors.